js/node/CMakeLists.txt

```diff
@@ -86,61 +96,29 @@ else()
 endif()
 
 if (WIN32)
-  target_link_libraries(onnxruntime_binding PRIVATE onnxruntime.lib)
+  target_link_libraries(onnxruntime_binding PRIVATE onnxruntime)
 elseif (APPLE)
   target_link_libraries(onnxruntime_binding PRIVATE libonnxruntime.${ort_version}.dylib)
```

Comment:
This line probably could also be changed to the same as the other platform.

```
target_link_libraries(onnxruntime_binding PRIVATE onnxruntime)
```
Then we can remove the ort_version variable.